rs/ethereum/cketh/mainnet/orchestrator_upgrade_2024_08_15.md

```diff
@@ -0,0 +1,42 @@
+```
+* [`0x1aBaEA1f7C830bD89Acc67eC4af516284b1bC33c`](https://etherscan.io/token/0x1aBaEA1f7C830bD89Acc67eC4af516284b1bC33c) is the address of the EURC smart contract on Ethereum Mainnet which can be verified on [EURC's website](https://developers.circle.com/stablecoins/docs/eurc-on-main-networks).
+* The transfer fee is `10_000`, corresponding approximately to 0.011 USD. This value was selected to be easy to reason about (1 followed by zeroes), while remaining within the acceptable range of 0.1 to 1 cent.
+
+
+## Release Notes
+
+No changes to the ckERC20 ledger suite orchestrator canister codebase, since this proposal uses the same version `3d0b3f10417fc6708e8b5d844a0bac5e86f3e17d` as the previous proposal ([131484](https://dashboard.internetcomputer.org/proposal/131484)).
+
+## Wasm Verification
+
+Verify that the hash of the gzipped WASM matches the proposed hash.
+
+```
+git fetch
+git checkout 3d0b3f10417fc6708e8b5d844a0bac5e86f3e17d
+./gitlab-ci/container/build-ic.sh -c
+sha256sum ./artifacts/canisters/ic-ledger-suite-orchestrator-canister.wasm.gz
```

Comment:
Can't verify the build with these commands, as the build fails with

```
FAILED: Build did NOT complete successfully (594 packages loaded, 14939 targets configured)
    currently loading: @crate_index__lzma-sys-0.1.20//
    Fetching repository @crate_index__icu_normalizer_data-1.5.0; starting
    Fetching repository @crate_index__icu_provider_macros-1.5.0; Restarting.
    Fetching repository @crate_index__writeable-0.5.5; Restarting.
    Fetching repository @crate_index__icu_locid-1.5.0; Restarting.
    Fetching https://static.crates.io/crates/zerofrom/0.1.4/download
    Fetching https://static.crates.io/crates/yoke/0.7.4/download
    Fetching https://static.crates.io/crates/zerovec-derive/0.10.3/download
ERROR: /ic/rs/sns/swap/proto_library/BUILD.bazel:24:13: in srcs attribute of rust_library rule //rs/sns/swap/proto_library:proto_library: source file '//rs/sns/swap/proto_library:src/gen' is misplaced here (expected .rs)
ERROR: /ic/rs/sns/swap/proto_library/BUILD.bazel:24:13: in srcs attribute of rust_library rule //rs/sns/swap/proto_library:proto_library: '//rs/sns/swap/proto_library:src/gen' does not produce any rust_library srcs files (expected .rs)
ERROR: /ic/rs/sns/swap/proto_library/BUILD.bazel:24:13: Analysis of target '//rs/sns/swap/proto_library:proto_library' failed
ERROR: Analysis of target '//publish/canisters:canisters' failed; build aborted:
```

Any idea what to do here?

Reply:
```
$ sha256sum ./artifacts/canisters/ic-ledger-suite-orchestrator-canister.wasm.gz
ccda8966249bc7d89f14968a242d6fb2c33d2363336303ec4f12977cab74c6f7  ./artifacts/canisters/ic-ledger-suite-orchestrator-canister.wasm.gz
```
Looks good!